### DISCUSSION TITLE
How do I resend a pending invitation?

### DISCUSSION THREAD

**Discussion by The-Futurist** [community/community#21682] [New to GitHub]
<p>I invited a person some time ago, they cant find the invitation email sent by GitHub, to me the invitation shows as pending - so how can I force a resend so that user gets another chance to receive the invitation email?</p>

**Comment by lee-dohm** [comment, marked as answer]
<p>You can find information on how to <a href="https://help.github.com/en/articles/canceling-or-editing-an-invitation-to-join-your-organization" rel="nofollow noopener">cancel or edit a pending invitation</a> in our help doucmentation. You should be able to cancel the invitation and then re-invite.</p>
<p>I hope that helps!</p>

**Reply by hn3000** [reply]
Sorry, is that really the suggested workflow in this case? Cancel the invitation and search for the person again?

**Reply by adamaviv** [reply]
Pinging on this request because this is a really silly and unhelpful workflow that just creates more work. Why can't we just resend the inviation?

**Reply by The-Futurist** [reply]
@lee-dohm - I agree with the others here, this seems to straightforward and obvious that it makes no sense to not at least put it in a "to do" list or queue. I'm sure there's _some_ work involved but the benefits are rather large, especially with large teams and frequently changing employees and collaborators.

**Reply by Jack-Ed** [reply]
What a joke of a response.

**Reply by Stewart-PN95** [reply]
The only option available as of  10-7-23 would be to remove the user from the organization or manage the user.  There is no invitation edit option.

**Reply by hn3000** [reply]
Yeah, still not a good response -- while a usable workaround, it's not ideal. Maybe have the devs add that button?

**Reply by Jack-Ed** [reply]
Indeed, @hn3000, Github isn't exactly an End of Life system. If a suggestion is made, it should be considered, given someone is willing to put the time into making the addition.

**Reply by Austen-EPIRA** [reply]
This seems like a basic need for growing teams. Please share updates if they come.

**Reply by tkralphs** [reply]
resending an invitation is a ridiculously convoluted process. It shouldn't require consulting documentation. The documentation is also out of date.
